From da4e8eaa0bf2829ffbea3991ecfb4f6c02676556 Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai.pt@gmail.com>
Date: Fri, 2 Aug 2024 17:29:42 +0800
Subject: [PATCH 1003/1005] ARM: dts: aspeed: catalina: add ipmb-dev node

The Catalina dts file was removing some properties and compatible strings which
are not support in db-binding document and cause `CHECK_DTBS=y` check failed.

ipmb-dev node was removed due to compatible string not exist on upstream.
Add back ipmb-dev node back in this patch.

Signed-off-by: Potin Lai <potin.lai.pt@gmail.com>
---
 .../boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts   | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
index 998be7b2a7c9..1ba886bf3177 100644
--- a/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
+++ b/arch/arm/boot/dts/aspeed/aspeed-bmc-facebook-catalina.dts
@@ -678,6 +678,16 @@ rtc@6f {
 
 &i2c7 {
 	status = "okay";
+	bus-frequency = <100000>;
+	multi-master;
+	aspeed,hw-timeout-ms = <1000>;
+
+	//USB Debug Connector
+	ipmb@10 {
+		compatible = "ipmb-dev";
+		reg = <(0x10 | I2C_OWN_SLAVE_ADDRESS)>;
+		i2c-protocol;
+	};
 };
 
 &i2c8 {
-- 
2.44.2

